src/main/groovy/org/gradle/android/AndroidCacheFixPlugin.groovy

```diff
@@ -90,8 +95,12 @@ class AndroidCacheFixPlugin implements Plugin<Project> {
     }
 
     /**
-     * Fix {@link org.gradle.api.tasks.compile.CompileOptions#getBootClasspath()} introducing relocatability problems for {@link AndroidJavaCompile}.
+     * The following are kept for reference purposes.
```

Comment:
I think Git history should be enough to keep these around.

Reply:
Yeah, if you know to look for it.  I left these around more for someone new to the plugin to have some examples of how to solve certain problems (beyond the few examples in the existing workarounds).  I don't feel strongly about it, and admittedly, it might be a bit much in terms of noise, but they're at least easier to discover this way.